app/views/citizens/additional_accounts/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<% content_for :navigation do %>
+  <%#= link_to 'Back', path-to-be-determined, class: 'govuk-back-link', id: 'back' %>
+<% end %>
+
+  <%= form_tag(citizens_additional_accounts_path) do %>
+
+    <div class="govuk-form-group">
+      <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+          <h1 class="govuk-fieldset__heading">
+            Do you have accounts with other banks?
+          </h1>
+        </legend>
+        <br />
+        <span class="govuk-hint">
+          Share details of any accounts you have with another bank or building society
+        </span>
+        <% if flash[:alert] %>
```

Comment:
I'm not entirely sure this is needed. The controllers are defining `flash[:error]` and the main layout [already has](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/ap-71-add-accounts-for-more-than-one-bank/app/views/layouts/application.html.erb#L60-L64) a general way of displaying those.